bliss/encoder/image_normalizer.py

```diff
@@ -106,17 +130,16 @@ def get_input_tensor(self, batch):
         if self.use_clahe:
             renormalized_img = self.clahe(raw_images, self.clahe_min_stdev)
             inputs.append(renormalized_img)
-            inputs[0] = self.clahe(backgrounds, self.clahe_min_stdev)
+            if not self.asinh_params:
+                inputs[0] = self.clahe(backgrounds, self.clahe_min_stdev)
 
         if self.asinh_params:
-            backgrounds = backgrounds.clamp(min=1e-6)
-            for threshold in self.asinh_params["thresholds"]:
-                inputs.append(
-                    torch.asinh(
-                        ((raw_images - backgrounds) / backgrounds.sqrt() - threshold)
-                        * self.asinh_params["scale"],
-                    )
+            inputs.append(
```

Comment:
Statements that span multiple lines using parentheses are hard to read. (Better not to rely on the `black` formatter to reformat long lines.) Can you define some intermediate variables instead?